packages/database/src/schema/models/promptSnaptshots.ts

```diff
@@ -1,4 +1,3 @@
-import { timestamps } from '$db/schema/schemaHelpers'
```

Comment:
since database package is imported as ts files into other projects typescript does not know how to interpret path aliases from other packages, resulting in type errors in those other packages. I've opted for removing path aliases in the database project in order to bypass this issue, but happy to hear any suggestions on how to solve this better.